### ISSUE TITLE
missing English common name for Coptidum ×spitsbergense; info on Nathorst's earlier name

### ISSUE THREAD

**Issue by sjmeades** [Canadensys/vascan-data#1736]
[Originally posted on GoogleCode (id 1713) on 2013-04-05]

<b>(This is the template to report a data issue for Vascan. If you want to</b>
<b>report another issue, please change the template above.)</b>

<b>What is the URL of the page where the problem occurs?</b>
http://data.canadensys.net/vascan/taxon/8568

<b>What data are incorrect or missing?</b>
missing English common name

<b>What data are you expecting instead?</b>
Svalbard buttercup

<b>If applicable, please provide an authoritative source.</b>
See: http://svalbardflora.net/index.php?id=131
Note, common name is along the right margin with other info.

Luc, you may be interested in the taxonomic information (below) from this site, and you may want to add Ranunculus pallasii Schltdl. var. spetsbergensis Nathorst as a synonym.  Also, Elven's name (used here), is a later isonym of Luferov and Probatova's Coptidium ×spitsbergense.

Two name forms have been applied: &quot;spitsbergensis&quot; (&quot;spitzbergensis&quot;) and &quot;spetsbergensis&quot;. When Hadač in 1942 published his name Ranunculus spitsbergensis,  he did it with a separate type and did not base it on Nathorst's name from 1883, var. spetsbergensis. Jalas (1988) showed that Nathorst's name, spelling and type have priority at varietal level, whereas Hadač' name, spelling and type have priority at species level.

**Comment by brouille** [comment]
[Originally posted on GoogleCode on 2013-04-08 13:10Z]

Slightly messy.Will have to sort this out.
Thanks. 
Luc

**Comment by brouille** [comment]
[Originally posted on GoogleCode on 2013-06-12 13:08Z]

Added the basionym and made some corrections to spelling fide IPNI and TROPICOS.

Changing the status from species to hybrid should not have changed the authorities. The name by Elven is nom. illeg. hom., it seems to me. I added it as such in syn.

Only the vernacular needs to be added now.
Marilyn, please take care of this issue.

Luc

**Comment by brouille** [comment]
[Originally posted on GoogleCode on 2013-06-12 13:11Z]

MArilyn

Forgot to add you in cc on this one, apparently. 

This morning, I cleaned up the Google-Code to see what I had to do still. In order to faciliate both my task and your own, I have now attributed all VernacularEN to you in Google-Code. It is now easy for you to select Owner (on the right of the field names bar) and select all issues that are yours only. This will make it clearer and simpler to handle remaining tasks.

Luc
